.github/pull_request_template.md

```diff
@@ -38,5 +38,5 @@ _Please confirm that if this PR changes any image versions, then that's the sole
 2. If no release note is required, just write "NONE".
 -->
 ```release-note
-
+ 
```

Comment:
looks great https://github.com/kubernetes-sigs/cluster-api-provider-azure/runs/5530753770?check_suite_focus=true

Go ahead and revert this doc change and this should be good to merge

Reply:
Done